x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/use_columns.tsx

```diff
@@ -150,7 +151,26 @@ export const useColumns = (
       ),
       width: '30px',
       render: (item) => {
-        return Array.isArray(item.alerting_rules) ? (
+        const needsReauth = needsReauthorization(item);
+
+        const needsReauthTooltipIcon = needsReauth ? (
+          <>
+            <EuiToolTip
+              content={i18n.translate(
+                'xpack.transform.transformList.needsReauthorizationBadgeTooltip',
+                {
+                  defaultMessage:
+                    'This transform was created with insufficient permissions. You must have manage_transform cluster privileges to reauthorize and start it.',
```

Comment:
@szabosteve @lcawl is it common practice to mention specific cluster privileges that are required? Wondering as in other places we aren't so specific:

<img width="268" alt="image" src="https://user-images.githubusercontent.com/7405507/232445339-ec59c769-c266-4d3b-97a4-c3ef84651359.png">

Reply:
After a quick check, I found that although we rarely mention specific cluster privileges in the UI, there is precedent for doing it, for example [in the APM app](https://github.com/elastic/kibana/blob/caa9231aef6a2b263ab6b9bfb121a1fe9fabfa1b/x-pack/plugins/apm/common/anomaly_detection/index.ts#L47) when you don't have the correct privileges to create anomaly detection jobs. While there is precedent, I would go without mentioning the privileges specifically. The docs mention the correct privileges and we consider them as the source of truth, so it might make sense to keep only those up-to-date – even if the name of the privileges is unlikely to change.
@peteharverson @lcawl WDYT?

```suggestion
                    'This transform was created with insufficient permissions. Contact your administrator to request the required privileges.',
```

Reply:
+1 to your suggestion @szabosteve to not mention the privileges specifically.

Reply:
Updated here  (https://github.com/elastic/kibana/pull/154736/commits/1a8969a95ca638855bb79e8a8e1b276a33ac2acc)

Reply:
Yup, that's even better, thanks!